app/views/patients/pledge.js.erb

```diff
@@ -0,0 +1,16 @@
+$('.modal-content').html('')
+$('.modal').attr('id', 'pledge-modal')
+$('.modal-content').append(
+  '<%= escape_javascript(render partial: "patients/pledge", locals: { patient: @patient, disable_next: disable_continue?(@patient) })%>'
+);
+
+$('.modal').modal('toggle');
```

Comment:
I added a new route for submitting pledge via ujs. The disabled next stuff is for the jquery multi pledge modal. Maybe we can roll it in a function, but this is pretty specific and i'm comfortable leaving as is until another modal comes around.